### PR TITLE
Always set canvas width and height

### PR DESCRIPTION
**Pull request by fabiomcosta**
Making sure we always set the canvas width/height so its properly sized on devices with pixelRatio=1.

### CODE REVIEW

File: packages/react-ape/renderer/reactApeRenderer.js [hunk 1]
```diff
@@ -256,12 +260,12 @@ const ReactApeFiber = reconciler({
 });
 
 const defaultContainer = {};
-const roots = new Map();
+// Using WeakMap avoids memory leak in case the container is garbage colected.
```
Comment (fabiomcosta) [inline packages/react-ape/renderer/reactApeRenderer.js]: bonus

File: packages/example-movie-list/src/App.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ class App extends React.Component {
         />
         <Text style={styles.heading}>• Netflix Originals</Text>
         <Text style={styles.time}>{this.state.time}</Text>
-        <Text style={styles.infoAboutRenderer}>Maded with React Ape</Text>
+        <Text style={styles.infoAboutRenderer}>Made with React Ape</Text>
```
Comment (fabiomcosta) [inline packages/example-movie-list/src/App.js]: bonus

File: packages/react-ape/renderer/components/ListView.js [hunk 3]
```diff
@@ -19,13 +19,8 @@ import React from 'react';
 */
 
 class ListView extends React.Component {
-  constructor(props) {
-    super(props);
-  }
-
```
Comment (fabiomcosta) [inline packages/react-ape/renderer/components/ListView.js]: the default constructor already achieves what you want

File: packages/react-ape/renderer/reactApeRenderer.js (hunk 1 above)
Comment (raphamorim) [reply packages/react-ape/renderer/reactApeRenderer.js]: 👏

**Pull request merged**